spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -1331,6 +1331,27 @@ The connector is responsible for the acknowledgment (positive or negative) of th
 * An outgoing connector must acknowledge the incoming `org.eclipse.microprofile.reactive.messaging.Message` once it has successfully dispatched the message.
 * An outgoing connector must acknowledge negatively the incoming `org.eclipse.microprofile.reactive.messaging.Message` if it cannot be dispatched.
 
+== Health
+
+When MicroProfile Reactive Messaging is used in an environment where MicroProfile Health is enabled, implicit readiness and liveness checks are produced.
+
+=== Readiness
```

Comment:
Please reword - such as "gets UP" should be reports `UP`.
Maybe a bit more precise:
- Readiness reports `DOWN` when messaging starts
- Once all subscribers.... the state changes to `UP`

Reply:
thx reworded